### PR TITLE
SNOW-1345147: Igarish snow 1345147 update quickstart guide

### PR DESCRIPTION
**Pull request by sfc-gh-igarish**
Python API supports managing resources like RBAC, Stages, and Service function. Anurag aks us to create a new quickstart guide for SPCS using Python API instead of SQL. I changed SQL to supported Python API. These Python APIs are in development right now and will available in QA deployment early next week. So please review it and then test using QA deployment next week. Once you guys approve this PR we will merge it.

### CODE REVIEW

**Review by sfc-gh-ghernandez — APPROVED** [review]

File: 00_setup.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        comment='My role to use container',
+    ))
+
+    # GRANT CREATE DATABASE ON ACCOUNT TO ROLE CONTAINER_USER_ROLE
+    # GRANT CREATE WAREHOUSE ON ACCOUNT TO ROLE CONTAINER_USER_ROLE;
+    # GRANT CREATE COMPUTE POOL ON ACCOUNT TO ROLE CONTAINER_USER_ROLE;
+    # GRANT CREATE INTEGRATION ON ACCOUNT TO ROLE CONTAINER_USER_ROLE;
+    # GRANT MONITOR USAGE ON ACCOUNT TO  ROLE  CONTAINER_USER_ROLE;
+    # GRANT BIND SERVICE ENDPOINT ON ACCOUNT TO ROLE CONTAINER_USER_ROLE;
+    root.grants.create(Grant(
```
Comment (sfc-gh-skumbham) [inline 00_setup.py:62]: `root.grants.grant`

File: 00_setup.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    # grant role CONTAINER_USER_ROLE to role ACCOUNTADMIN;
+    root.grants.create(Grant(
+        grantee=Grantees.role('ACCOUNTADMIN'),
+        securable=Securables.role('CONTAINER_USER_ROLE'),
+        privileges=[Privileges.usage],
+    ))
+
+    # Connect as CONTANTAINER_USE_ROLE
+    connection_container_user_role = connect(**CONNECTION_PARAMETERS_CONTAINER_USER_ROLE)
```
Comment (sfc-gh-skumbham) [inline 00_setup.py:90]: update this to root.session.use_role()

File: 00_setup.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+#!/opt/conda/bin/python3
+import os
+
+from snowflake.core import Root
+from snowflake.core._common import CreateMode
+from snowflake.core.warehouse import Warehouse
+from snowflake.core.stage import Stage
+
+from snowflake.core.grants import (
+    DeletionMode,
```
Comment (sfc-gh-skumbham) [inline 00_setup.py:11]: DeletionMode is not used

File: 00_setup.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    Grant,
+    Grantees,
+    Privileges,
+    Role,
```
Comment (sfc-gh-skumbham) [inline 00_setup.py:15]: Remove Role from here. Role is imported from `from snowflake.core.role import Role`

**Pull request merged**